lidar_frame_id: "uav1/os_lidar" # Name of the sensor frame
global_frame_id: "uav1/gps_origin" # Name of the map origin frame

# lidar_topic: "/uav1/pcl_filter_ouster/points_processed" # LiDAR sensor topic
lidar_topic: "/uav1/os_cloud_nodelet/points" # LiDAR sensor topic
# lidar_topic: "/uav1/pcl_filter/points_processed" # LiDAR sensor topic

odom_topic: "/uav1/mavros/global_position/local"
octomap_vis_topic: "/occupied_octomap_cells" # Visual octomap topic
# pcd_path: "/home/ivan/user_ros_workspace/src/basic.pcd" # Leave empty if you're working with a sensor pointcloud
pcd_path: "" # Leave empty if you're working with a sensor pointcloud



# Exploration depth of octomap tree. 16 is best resolution,
# slowest exploration
# explorationDepth: 14 this is not being used, its being hardcoded, fix this
radius: 2.5

# --- Octomap parameters
resolution: 0.5 # [m]
# prob_hit: 0.7 #
# prob_miss: 0.4 #
prob_hit: 0.96 #
prob_miss: 0.47 #
clamping_thresh_min: 0.12 #
clamping_thresh_max: 0.97 #
# ---

inflation_radius: 0.5 # [m] Set 0 to ignore inflation
ground_removal_dist: 0.0 # [m] Set 0 to ignore ground removal

# --- Does not work when a pcd_path is available
min_sensor_range: 1.0 # [m]
max_sensor_range: 20.0 # [m]
# ---

#--- Frontier exploration parameters
# Imaginary bounding box to restrict the exploration
bbx_minX: -38
bbx_maxX: 38
bbx_minY: -12
bbx_maxY: 13.0
bbx_minZ: 0
bbx_maxZ: 7.0

kernel_bandwidth: 0.5 #cambie esto de 1 a 2
kGain: 100.00
lambda: 0.1386
boxInfGainSize: 5.0


